ansible/templates/minigraph_dpg.j2

```diff
@@ -180,7 +180,7 @@
         <IPInterface>
           <Name i:Name="true"/>
 {% if 'port-channel' in vm_topo_config['vm'][vms[index]]['ip_intf'][dut_index|int]|lower %}
-          <AttachTo>PortChannel{{ ((index+1) |string).zfill(4) }}</AttachTo>
+          <AttachTo>PortChannel{{ ((index+1) |string).zfill(2) }}</AttachTo>
```

Comment:
what is need of this change ?

Reply:
@sanmalho-git i think you might not have this change https://github.com/Azure/sonic-mgmt/pull/4576.
After above PR External Portchannel will always have 4 digit. Can you revert this particular change and then we can merge the PR. I have verified other change is good.

Reply:
@sanmalho-git i think you might not have this change #4576.
After above PR External Portchannel will always have 4 digit. Can you revert this particular change and then we can merge the PR. I have verified other change is good.

Reply:
Yes - I missed that PR in my testing. I have reverted the commit that changed the PortChannels to have 2 digits instead of 4.